### PR TITLE
CI action

### PR DESCRIPTION
**Pull request by MCJOHN974**
@yegor256 i add github job which builds and commits objects

### CODE REVIEW

**Comment by yegor256** [comment]
@MCJOHN974 adding a binary file to Git is a bad idea. During CI cycle we should just do `make` and that's it. You may add another GitHub action, which will package the product and create a new GitHub release. This action should be triggered by a new tag added to Git.

**Comment by MCJOHN974** [comment]
@yegor256 so, this job should only check that build works without errors/warnings, and shouldn't save executable anywere ?

**Comment by yegor256** [comment]
@MCJOHN974 that's right. CI is only about running the build and checking that it's "green". That's it. Also, would be great to add a badge to README.md.

**Comment by MCJOHN974** [comment]
@yegor256 now CI job only check build, i tested it works with "green" and "red" builds both

File: .github/workflows/c-cpp.yml [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+  build:
+
+    runs-on: ubuntu-latest
+
+    steps:
+    - uses: actions/checkout@v2
+    - name: to_carts_repo
+      run: cd ../../
+    - name: make_objects
+      run: make ./bin/objects.a
```
Comment (yegor256) [inline .github/workflows/c-cpp.yml:17]: @MCJOHN974 maybe just `make` is enough?

File: .github/workflows/c-cpp.yml [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+
+jobs:
+  build:
+
+    runs-on: ubuntu-latest
+
+    steps:
+    - uses: actions/checkout@v2
+    - name: to_carts_repo
+      run: cd ../../
```
Comment (yegor256) [inline .github/workflows/c-cpp.yml]: @MCJOHN974 I believe this is redundant. Try to remove lines 16 and 17 and you will see that nothing changes.

**Comment by yegor256** [comment]
@MCJOHN974 see comments above

**Comment by MCJOHN974** [comment]
@yegor256 16 and 17 lines are really useless, but just ```make``` instead of ```make ./bin/objects.a``` doesn't work with my ```Makefile```. I remove 16 and 17 lines in last commit.

**Comment by yegor256** [comment]
@MCJOHN974 thanks

**Pull request merged**